jme3-core/src/plugins/java/com/jme3/material/plugins/ShaderNodeLoaderDelegate.java

```diff
@@ -660,19 +690,13 @@ public VariableMapping readInputMapping(Statement statement1) throws IOException
                 throw new MatParseException("Could not find a Material Parameter named " + right.getName(), statement1);
             }
             if (shaderNode.getDefinition().getType() == Shader.ShaderType.Vertex) {
-                if (updateRightFromUniforms(param, mapping, vertexDeclaredUniforms, statement1)) {                  
+                if (updateRightFromUniforms(param, mapping, vertexDeclaredUniforms, statement1)) {
+                    updateMaterialTextureType(statement1, mapping, left, param);

```

Comment:
mhhh I'm not sure vertex texture fetch is available de facto with opengl2.0 or opengles 1.0.
So we may need to check this at generation time. and warn the user and provide a way to fallback to another solution if relevant...

Reply:
@Nehon but in any case, you can't do it with any openGL version, maybe do you have a better idea how to allow it for users?

Reply:
Well... this one is though... because we can only know when we load the definition...
I guess keep it like this. And let's leave the check to the user willing to use this feature...